browse-scaler/src/cmr.js

```diff
@@ -1,13 +1,9 @@
-const fetch = require('isomorphic-unfetch');
+const fetch = require('node-fetch');
 
 /* CMR ENVIRONMENT VARIABLES */
-const environment =
-  process.env.CMR_ENVIRONMENT !== 'sit' && process.env.CMR_ENVIRONMENT !== 'uat'
-    ? ''
-    : `${process.env.CMR_ENVIRONMENT}.`;
-const cmrRootUrl = `https://cmr.${environment}earthdata.nasa.gov/search/`;
-const cmrCollectionUrl = `${cmrRootUrl}collections.json?concept_id=`;
-const cmrGranuleUrl = `${cmrRootUrl}granules.json?collection_concept_id=`;
+const cmrRootUrl = process.env.CMR_ROOT;
+const cmrCollectionUrl = `${cmrRootUrl}/search/collections.json?concept_id=`;
+const cmrGranuleUrl = `${cmrRootUrl}/search/granules.json?concept_id=`;
 
 const fetchConceptFromCMR = async (conceptId, cmrEndpoint) => {
   return fetch(cmrEndpoint + conceptId)
```

Comment:
Should there be error handling here? What happens if the concept id does not exist then there would be no `entry[0]`

Reply:
It _shouldn't_ bite us because edsc won't be sending a whole lot of concept-ids that don't return search results, but it's not a bad idea to add a try/catch here, so I will :)